lib/components/src/form/button.js

```diff
@@ -0,0 +1,12 @@
+import glamorous from 'glamorous';
+
+export default glamorous
+  .button({
+    border: 'solid 1px rgba(0, 0, 0, 0.2)',
+    background: 'rgba(255, 255, 255, 0.5)',
+    padding: '5px 10px',
+    borderRadius: '4px',
+    color: 'rgba(0, 0, 0, 0.5)',
```

Comment:
It looks disabled to me as it is now:
<img width="91" alt="screen shot 2017-11-11 at 03 54 08" src="https://user-images.githubusercontent.com/6651625/32684400-309dcf82-c694-11e7-9eba-3dd5301517fa.png">